src/client-side-encryption/auto_encrypter.ts

```diff
@@ -285,6 +289,10 @@ export class AutoEncrypter {
       mongoCryptOptions.bypassQueryAnalysis = options.bypassQueryAnalysis;
     }
 
+    if (options.keyExpirationMS) {
```

Comment:
Passing `0` to disable caching entirely is valid, right? So

```suggestion
    if (options.keyExpirationMS != null) {
```